etc/api/BUILD

```diff
@@ -36,14 +36,16 @@ load("@npm_ts_api_guardian//:index.bzl", "ts_api_guardian_test")
     ),
     # We don't want to analyse these exports nor add them to the golden files
     # in most cases it's because Ts API Guardian doesn't support Symbol Aliases.
+    # Note: $ must be escaped to $$ because these patterns will be executed by shell
+    # and $ has a special meaning there.
     strip_export_pattern = [
         # @angular-devkit/architect
-        "^BuilderProgressState$",
+        "^BuilderProgressState$$",
         # @angular-devkit/schematics
-        "^workflow$",
-        "^formats$",
+        "^workflow$$",
+        "^formats$$",
         # @angular-devkit/build-optimizer
-        "^buildOptimizerLoader$",
+        "^buildOptimizerLoader$$",
```

Comment:
This replacement might need to be done in `ts_api_guardian` itself, as it already contains one of these replacements for shell compat in https://github.com/angular/angular/blob/08348fc2e8b48d58a30063b0b5b03d21158a315b/tools/ts-api-guardian/index.bzl#L111-L113.

The change seems to be related to https://github.com/bazelbuild/rules_nodejs/commit/5358d56.

Reply:
Discussed with @gregmagolan, he has a better idea of where to fix this.

Reply:
I believe I landed the fix for this in 1.6.0